QuasarHelios2015/src/org/team1540/quasarhelios/Clamp.java

```diff
@@ -80,6 +87,21 @@ public static void setup() {
         Cluck.publish("Clamp CAN Bus Voltage Fault", BooleanMixing.createDispatch(clampCAN.getDiagnosticChannel(ExtendedMotor.DiagnosticType.BUS_VOLTAGE_FAULT), updateCAN));
         Cluck.publish("Clamp CAN Temperature Fault", BooleanMixing.createDispatch(clampCAN.getDiagnosticChannel(ExtendedMotor.DiagnosticType.TEMPERATURE_FAULT), updateCAN));
 
+        BooleanInputPoll maxCurrentNow = FloatMixing.floatIsAtLeast(clampCAN.asStatus(ExtendedMotor.StatusType.OUTPUT_CURRENT),
+                ControlInterface.mainTuning.getFloat("Clamp Max Current Amps +M", 45));
+        EventInput maxCurrentEvent = EventMixing.filterEvent(maxCurrentNow, true, Igneous.constantPeriodic);
+
+        QuasarHelios.publishStickyFault("clamp-current-fault", maxCurrentEvent);
+
+        FloatStatus clampResetTime = ControlInterface.mainTuning.getFloat("Clamp Reset Time", 1.0f);
+
+        maxCurrentEvent.send(EventMixing.filterEvent(enabled, true, () -> {
+            enabled.set(false);
+            ExpirationTimer timer = new ExpirationTimer();
+            timer.schedule(clampResetTime, enabled.getSetTrueEvent());
+            timer.start();
```

Comment:
No. As a reminder, you never create mixing objects within any event. Just set up the timer outside and then use `startWhen`, `startOrFeedWhen`, or `getStartEvent`, or `getStartOrFeedEvent`.